pkgs/applications/audio/ams/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchgit
+, automake
+, alsaLib
+, ladspaH
+, libjack2
+, fftw
+, zita-alsa-pcmi
+, qt5
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "ams";
+  version = "unstable-2019-04-27";
+
+  src = fetchgit {
+    url = https://git.code.sf.net/p/alsamodular/ams.git;
+    sha256 = "0qdyz5llpa94f3qx1xi1mz97vl5jyrj1mqff28p5g9i5rxbbk8z9";
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook qt5.wrapQtAppsHook ];
+
+  buildInputs = with qt5; [ alsaLib ladspaH libjack2 fftw zita-alsa-pcmi qtbase qttools ];
```

Same here. Plus it is a good practice to keep the scope of the with expression as narrow as possible.

```suggestion
  buildInputs = [
    alsaLib
    ladspaH
    libjack2
    fftw
    zita-alsa-pcmi
  ] ++ (with qt5; [
    qtbase
    qttools
  ]);
```